static/src/stylesheets/module/_social.scss

```diff
@@ -159,6 +165,13 @@ category: Common
     }
 }
 
+.social-icon--messenger {
+    background-color: $social-messenger;
+    &:hover {
+        background-color: darken($social-messenger, 10%);
```

Comment:
Is this, erm, official Facebook brand hover colour..? Also, you might want to add `:focus` and `:active` states here.